tools/maptools.py

```diff
@@ -69,13 +72,17 @@ def __deploy__(self):
     def __copy__(self):
         mapcopy(self.args)
 
+    def __test__(self):
```

Comment:
Hm, it's wrong to make this a dunder method, since dunder methods are supposed to be reserved for the language. Still, the rest of the class does it like that, so for consistency it's good to do the same.